docs/sources/datasources/pyroscope/configure-pyroscope-data-source.md

```diff
@@ -42,16 +42,85 @@ To configure basic settings for the data source, complete the following steps:
-   | `Minimal step` | Used for queries returning timeseries data. The Pyroscope backend, similar to Prometheus, scrapes profiles at certain intervals. To prevent querying at smaller interval, use Minimal step same or higher than your Pyroscope scrape interval. This prevents returning too many data points to the frontend. |
+## Configuration options
+
+You can configure several options for the Pyroscope data source, including the name, HTTP, authentication, querying, and private data source connect.
+
+If you make any changes, select **Save & test** to preserve those changes.
+
+![Configuration options for the Pyroscope data source](/media/docs/grafana/data-sources/screenshot-pyroscope-data-source-config.png)
+
+### Name and default
+
+**Name**
+: Enter a name to specify the data source in panels, queries, and Explore.
+
+**Default**
+: The default data source is pre-selected for new panels.
+
+### HTTP
+
+The HTTP section is shown in number 1 in the screenshot.
+
+**URL**
+: The URL of the Grafana Pyroscope instance, for example, `https://localhost:4100`.
+
+**Allowed cookies**
+: The Grafana Proxy deletes forwarded cookies. Use this field to specify cookies by name that should be forwarded to the data source.
+
+**Timeout**
+: HTTP request timeout in seconds.
+
+### Auth
+
+The Auth section is shown in number 2 in the screenshot.
+
+**Basic auth**
+: Enable basic authentication to the data source. When activated, it provides **User** and **Password** fields.
+
+**With Credentials**
+: Whether credentials, such as cookies or auth headers, should be sent with cross-site requests.
+
+**TLS Client Auth**
+: Toggle on to use client authentication. When enabled, it adds the `Server name`, `Client cert` and `Client key`. The client provides a certificate that is validated by the server to establish the client's trusted identity. The client key encrypts the data between client and server. These details are encrypted and stored in the Grafana database.
+
+**With CA Cert**
+: Activate this option to verify self-signed TLS certificates.
+
+**Skip TLS Verify**
+: When activated, it bypasses TLS certificate verification.
+
+**Forward OAuth Identity**
+: When activated, the user’s upstream OAuth 2.0 identity is forwarded to the data source along with their access token.
+
+**Custom HTTP Headers**
+: Select Add header to add Header and Value fields.
+
+**Header**
+: Add a custom header. This allows custom headers to be passed based on the needs of your Pyroscope instance.
+
+**Value**
+: The value of the header.
+
+### Querying
+
+The **Querying** section is shown in number 3 in the screenshot.
+
+**Minimum step** is used for queries returning time-series data. The default value is 15 seconds.
+
+The Pyroscope backend, similar to Prometheus, scrapes profiles at certain intervals. To prevent querying at smaller interval, use Minimal step same or higher than your Pyroscope scrape interval. This prevents returning too many data points to the frontend.
```

Comment:
The equivalent setting in Prometheus is https://github.com/grafana/grafana/blame/a16782a2d6c5823dd8d1abdc463da9f8d8b8113a/docs/sources/datasources/prometheus/configure-prometheus-data-source.md#L112. Unfortunately the name of the setting and therefore the  doc page is not very consistent with how it is done there.

In general the scrape interval is not configured in the Pyroscope backend and rather either in the SDKs or in Grafana Agent or Alloy. It basically is the value how often profiles would be typically collected. When comparing with grafana alloy it is called the "scrape_interval" there.

This is therefore a wrong statement: `The Pyroscope backend, similar to Prometheus, scrapes profiles at certain intervals.` as Pyroscope never scrapes.

`This prevents returning too many data points to the frontend`, that might be true, but the actual reason for this parameters is to prevent gaps when you zoom in too much. (As reported here: https://github.com/grafana/explore-profiles-deprecated/issues/399).

Reply:
Thank you for the clear explanation. I think I pulled that explanation from the Prometheus docs, and wasn't sure if ti was correct or not. In the previous version of the data source screen (used for this doc), the additional information in the hover text for Minimum step said: 
![image](https://github.com/grafana/grafana/assets/104772500/904027b0-5c6b-4a1e-bc12-65515dd1ed9a)

The screen has been updated and the Querying section is actually removed from the data source, so this information can be removed.

Reply:
Actually, it's not removed from the data source. It just wasn't present in the data source that I was looking at. .

Reply:
What if we minimize this information? How is this? 
@simonswine 

```suggestion
**Minimum step** is used for queries returning time-series data. The default value is 15 seconds.

Adjusting this option can help prevent gaps when you zoom in to profiling data. 
```

Reply:
That's perfect